README.md

```diff
@@ -0,0 +1,130 @@
+# Understanding a Temporal workflow as an implementation of the Actor Model
+
+The project demonstrates how key features of the Actor Model map onto Temporal Workflows.
+
+| NOTE                                                                                                                                                                                                                                                   |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| In order to get full benefit from using this project, it's useful to have a basic understanding of what Temporal is and how it works. [This tutorial](https://docs.temporal.io/temporal) provides a good basic introduction to the Temporal framework. |
+
+
+In the context of the Actor Model, the sample project treats a [Temporal Workflow](https://docs.temporalio/workflows)
```

Comment:
Should the Actor Model be defined in this README?  Or is it expected that the reader already knows what the Actor Model is?

Reply:
Added a link to the term "Actor Model" that points to a Temporal blog article that describes the Actor Model:

addressed

Reply:
OK